scripts/publish.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -e
+
+MSG_PUBLISH_SUCCESS="npm publish: Succeed"
+MSG_PUBLISH_FAIL="npm publish: Failed"
+
+echo "--------------------------------------------"
+echo "|    Publishing has been triggered          |"
+echo "--------------------------------------------"
+
+echo "travis branch    -> $TRAVIS_BRANCH"
+echo "travis PR branch -> $TRAVIS_PULL_REQUEST_BRANCH"
+git status
+
+set_git_remote() {
+    git remote set-url origin https://${GH_ACCESS_TOKEN}@github.com/capsulajs/environment-registry
+}
+
+echo_result() {
+    if [[ "$1" == 0 ]]; then
+        echo "$MSG_PUBLISH_SUCCESS" && return 0
+    else
+        echo "$MSG_PUBLISH_FAIL" && return 1
+    fi
+}
+
+if [[ "$TRAVIS_PULL_REQUEST_BRANCH" =~ ^feature\/.*$ ]]; then
+    echo "--------------------------------------------"
+    echo "|    Deploying 'snapshot' on npm registry    |"
+    echo "--------------------------------------------"
+
+    BRANCH_NAME=$(echo $TRAVIS_PULL_REQUEST_BRANCH | sed "s/[_/]/-/g")
+    PACKAGE_VERSION="alpha.$(date +%s).$BRANCH_NAME"
+    npm version prerelease --preid="$PACKAGE_VERSION"
+    git push origin $BRANCH_NAME
```

Comment:
This won't work as you don't do `set_git_remote` before.  
Anyway, I am not sure that it is a good idea to push on the feature branch during the CI. It will only add pollution